lib/toy/behaviours.rb

```diff
@@ -25,5 +25,36 @@ def remove_instance_variable(name)
         ivar_map.delete(name.to_sym)
       end
     end
+
+    module Constants
+      def inspect
+        "#<#{self.class}>"
+      end
```

Comment:
Yep, totally fair! I was considering it a requirement of those constant methods rather than a part of the Object API, but it makes more sense to treat this as part of the Ruby API that we're trying to implement in our Toy object space 👍